providers/edge/src/airflow/providers/edge/example_dags/win_notepad.py

```diff
@@ -19,7 +19,7 @@
 In this DAG is a demonstrator how to interact with a Windows Worker via Notepad.
 
 The DAG is created in conjunction with the documentation in
-https://github.com/apache/airflow/blob/main/docs/apache-airflow-providers-edge/install_on_windows.rst
+https://github.com/apache/airflow/blob/main/airflow-core/docs-providers-edge/install_on_windows.rst
```

Comment:
I am not sure but seems like this URL replacement is "wrong"?

Reply:
👀

Reply:
Yeah. the original URL has been wrong - we did not have docs/apache-airflow-providers-edge/ any more - we should bave /providers/apache-airflow-providers-edge/docs/  -> that's why my search replace did what it did.

Reply:
```https://github.com/apache/airflow/blob/main/providers/edge/docs/install_on_windows.rst`` -> that's the right URL

Reply:
Thanks for fixing!